### ISSUE TITLE
The swipe down gesture barely works and constantly gets stuck half way

### ISSUE THREAD

**Issue by ell1e** [maliit/keyboard#223]
See here:

[100_5457.webm](https://github.com/maliit/keyboard/assets/64124388/d0e49f3e-9102-45af-a627-37b044e1f6e5)

It would help if there was [an obvious, additional alternative like a hide button](https://github.com/maliit/keyboard/issues/222).

**Comment by ell1e** [comment]
Is there any emergency workaround available for this?

I've been trying to use this device more seriously over some hours, and this issue single-handedly causes RSI flare ups and generally makes navigating any website a huge pain for me (because input fields will cause maliit to come up which is fine, but then I'm literally hurting my wrist and fingers trying to get it out of the way).

**Comment by espidev** [comment]
Yes: https://github.com/maliit/keyboard/pull/211

**Comment by ell1e** [comment]
Thanks so much! So building from that pull request's version might help? I'll try it right now! :tada: :star:

**Comment by espidev** [comment]
Yes, it should address the issue, but please let me know if there are problems with the PR

**Comment by ell1e** [comment]
My apologies for perhaps just being confused, but are there any build instructions available? I searched in the repository and can't find any in `README.md`, and I can't see any `BUILD.md` either. I tried building it after installing the Qt and QML development headers for Qt5 and Qt6, but cmake complains that I need some "MaliitPlugins". It's not entirely clear to me however why they are needed, when the "plugins" folder is right there.

**Comment by espidev** [comment]
Install the packages from the makedepends here: https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/maliit-keyboard/APKBUILD

Then in the folder, do 

```
mkdir build
cd build
cmake ..
make -j4
sudo make install
```

**Comment by ell1e** [comment]
Thanks to your instructions, I successfully built the latest git dev version :heart: and verified that it works. Since it fixes #225 that's already a huge improvement for me. I will now rebuild with the changes of the pull request on top and report back on the state of this issue here.

**Comment by ell1e** [comment]
I posted the test results here: https://github.com/maliit/keyboard/pull/211#issuecomment-2217778560

**Comment by espidev** [comment]
Thanks, I'll try to take a look soon
